apps/src/templates/sectionProgress/progressTables/ProgressTableView.jsx

```diff
@@ -136,29 +137,72 @@ class ProgressTableView extends React.Component {
     }
   }
 
-  // override the default initial number of rows to render
+  /**
+   * Override the default initial number of rows to render
+   */
   setRowsToRender() {
     const initialRows = parseInt(progressTableStyles.MAX_ROWS);
 
     // amountOfRowsToRender is a reactabular internal
-    this.studentList &&
-      this.studentList.bodyComponent.setState({
-        amountOfRowsToRender: initialRows
-      });
-    this.contentView &&
-      this.contentView.bodyComponent.setState({
-        amountOfRowsToRender: initialRows
-      });
+    this.studentList?.bodyComponent.setState({
+      amountOfRowsToRender: initialRows
+    });
+    this.contentView?.bodyComponent.setState({
+      amountOfRowsToRender: initialRows
+    });
+    this.syncScrollTop();
   }
 
   onScroll(e) {
     this.studentList.body.scrollTop = e.target.scrollTop;
     this.contentView.header.scrollLeft = e.target.scrollLeft;
+    this.scrollTop = e.target.scrollTop;
```

Comment:
the crux of this change is that we now store the target `scrollTop` during scrolling so we can force synchronization when scrolling finishes, view type changes, or rows are expanded/collapsed.